### PR TITLE
DOC-4573: Updates to remove partition ranges and parameters

### PR DESCRIPTION
**Pull request by beajohnson**
Working with @pravinbhat and @msmygit, I will remove the Migrate or validate specific partition ranges section, and ensure the Use Cassandra Data Migrator Link section has the correct GH link. I’ll also delete the CDM parameters section as there are links available to the parameters in GH.

### CODE REVIEW

**Comment by plpesvc-ds** [comment]
Antora site build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Antora site build successful! :white_check_mark:
Deploying draft.

**Comment by msmygit** [comment]
@beajohnson with `4.3.5` we have also updated the Spark version from `3.5.1` to `3.5.2`, so https://d5rxiv0do0q3v.cloudfront.net/DOC-4573/data-migration/cassandra-data-migrator.html#cdm-prerequisites needs to change as well.

**Comment by plpesvc-ds** [comment]
Antora site build successful! :white_check_mark:
Deploying draft.

**Review by msmygit — APPROVED** [review]
LGTM 👍🏼

**Comment by plpesvc-ds** [comment]
The draft build for this pull request has been cleaned up due to being stale.

Draft builds 2 weeks old or older were automatically removed.

You can rebuild the draft at any time by pushing a new commit to this pull request:

```
git checkout DOC-4573
git commit --allow-empty -m 'rebuild draft'
git push origin DOC-4573
```

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

File: modules/ROOT/partials/cdm-prerequisites.adoc [hunk 1]
```diff
@@ -2,15 +2,15 @@ Read the prerequisites below before using the Cassandra Data Migrator.
 
 * Install or switch to Java 11.
 The Spark binaries are compiled with this version of Java.
-* Select a single VM to run this job and install https://archive.apache.org/dist/spark/spark-3.5.1/[Spark 3.5.1] there.
-No cluster is necessary.
-* Optionally, install https://maven.apache.org/download.cgi[Maven] 3.9.x if you want to build the JAR for local development.
+* Select a single VM to run this job and install https://archive.apache.org/dist/spark/spark-3.5.3/[Spark 3.5.3] there.
+No cluster is necessary and is recommended for most one-time migrations however cluster mode is also supported for complex migrations.
```
Comment (KimberlyFields) [inline modules/ROOT/partials/cdm-prerequisites.adoc:6]: This sentence is unclear. 

"No cluster is necessary" - that part is clear. 

"and is recommended for most one-time migrations" - what is recommended? Not having a cluster? 

"however cluster mode is also supported for complex migrations" - how is this information related to the first part of the sentence, "no cluster is necessary"?

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -41,7 +41,7 @@
 * {cstar-data-migrator}
 ** xref:cdm-overview.adoc[]
 ** xref:cdm-steps.adoc[Migrate data]
-** xref:cdm-parameters.adoc[Parameters]
+//** xref:cdm-parameters.adoc[Parameters]
```
Comment (KimberlyFields) [inline modules/ROOT/nav.adoc:44]: Why is this being commented out instead of removed?

Comment (msmygit) [reply modules/ROOT/nav.adoc]: Now completed remove that file along with other partial files too.

File: modules/ROOT/partials/cdm-prerequisites.adoc (hunk 1 above)
Comment (msmygit) [reply modules/ROOT/partials/cdm-prerequisites.adoc]: Clarified the sentence for the spark cluster requirement.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build failure! :x:
> {"level":"error","time":1734105645355,"name":"asciidoctor","file":{"path":"modules/ROOT/pages/cdm-steps.adoc","line":18},"source":{"url":"https://github.com/datastax/migration-docs.git","refname":"DOC-4573","reftype":"branch"},"msg":"target of include not found: partial$cdm-partition-ranges.adoc"} > tailwindcss > tailwindcss build -c ${npm_config_tailwind_config_path} -i ${npm_config_css_path} -o ${npm_config_css_path} --minify Browserslist: caniuse-lite is outdated. Please run: npx update-browserslist-db@latest Why you should do it regularly: https://github.com/browserslist/update-db#readme Rebuilding... Done in 2590ms.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Review by KimberlyFields — APPROVED** [review]
